geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HashExecutor.java

```diff
@@ -42,14 +41,14 @@
    * @param key the region hash key region:<key>
    * @return the map data
    */
-  protected Map<ByteArrayWrapper, ByteArrayWrapper> getMap(ExecutionHandlerContext context,
+  protected RedisHash getMap(ExecutionHandlerContext context,
```

Comment:
can we deprecate this method?

Reply:
We could but we still will use it internally until the other HASH commands are converted to use functions. I don't think it is worth marking the internal apis we need to support this older implementation as deprecated. We should just spend a day converting the other HASH commands to use a function and then we can get rid of this, and other, methods.